### PR TITLE
Adding --downtime-replicas argument, following up pull/38 issues/22

### PR DESCRIPTION
**Pull request by grebois**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -106,6 +106,7 @@ Available command line options:
     Exclude specific deployments from downscaling (default: kube-downscaler, downscaler), can also be configured via environment variable ``EXCLUDE_DEPLOYMENTS``
 ``--exclude-statefulsets``
     Exclude specific statefulsets from statefulsets, can also be configured via environment variable ``EXCLUDE_STATEFULSETS``
+``--downtime-replicas``
```
Comment (hjacobs) [inline README.rst]: Please add one sentence as documentation

File: README.rst [hunk 2]
```diff
@@ -34,7 +34,7 @@ It will scale down the deployment's replicas if all of the following conditions
 * the deployment is not marked for exclusion (annotation ``downscaler/exclude: "true"``)
 * there are no active pods that force the whole cluster into uptime (annotation ``downscaler/force-uptime: "true"``)
 
-The deployment by default will be scaled down to zero replicas. This can be configured with a deployment annotation of ``downscaler/downtime-replicas``. (eg: ``downscaler/downtime-replicas: "1"``)
+The deployment by default will be scaled down to zero replicas. This can be configured with a deployment annotation of ``downscaler/downtime-replicas`` (eg: ``downscaler/downtime-replicas: "1"``) or via cli with ``--downtime-replicas``.
```
Comment (hjacobs) [inline README.rst]: eg -> e.g.
cli -> CLI

**Comment by hjacobs** [comment]
Thanks, can you add a test case for the new option?

File: README.rst (hunk 1 above)
Comment (grebois) [reply README.rst]: Updated

File: README.rst (hunk 2 above)
Comment (grebois) [reply README.rst]: Updated

**Comment by grebois** [comment]
Added test

File: kube_downscaler/cmd.py [hunk 3]
```diff
@@ -28,4 +28,7 @@ def get_parser():
     parser.add_argument('--exclude-statefulsets',
                         help='Exclude specific statefulsets from downscaling',
                         default=os.getenv('EXCLUDE_STATEFULSETS', ''))
+    parser.add_argument('--downtime-replicas',
```
Comment (hjacobs) [inline kube_downscaler/cmd.py]: should be of type `int`

Comment (hjacobs) [reply kube_downscaler/cmd.py]: fixed

**Comment by hjacobs** [comment]
Thanks!

**Comment by grebois** [comment]
Thanks @hjacobs

**Pull request merged**